Snakefile

```diff
@@ -337,3 +338,19 @@ rule sysbio:
           --outputFile {output.evidenceFile}
         opentargets_validator --schema {params.schema} {output.evidenceFile}
         """
+
+# --- Data sources parsers --- #
```

Comment:
If we want to separate the rules per data type, I would change the text above to `# --- Evidence data sources parsers --- #`.

The distinction is still not super accurate because `phenodigm` outputs the Mouse Phenotypes dataset as well.

```suggestion
# --- Target annotation data sources parsers --- #
```